src/ontology/mondo-edit.obo

```diff
@@ -226905,7 +226897,6 @@ xref: Orphanet:64755 {source="MONDO:equivalentTo", source="OMIM:604919"}
 xref: UMLS:C0263579 {source="Orphanet:64755"}
 xref: UMLS:C1858042 {source="Orphanet:64755", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:604919"}
 is_a: MONDO:0005073 {source="Orphanet:64755"} ! melanocytic nevus
-is_a: MONDO:0015853 {source="Orphanet:64755"} ! deficient breast volume or number
 is_a: MONDO:0015950 {source="Orphanet:64755"} ! inherited skin tumor
 is_a: MONDO:0019755 ! developmental defect during embryogenesis
```

Comment:
this source is not correctly formatted

Reply:
yeah, true. There are a million things to clean up in Mondo, so I haven't been doing all the things. We do have a ticket to address these unsupported superclass axioms, so we'll get to it at some point:
https://github.com/monarch-initiative/mondo/issues/5857

The ticket says remove unsupported associations but we will review them first.